pkg/endpoint/policy.go

```diff
@@ -450,7 +527,9 @@ func (e *Endpoint) regenerateL3Policy(owner Owner, repo *policy.Repository, revi
 // IngressOrEgressIsEnforced returns true if either ingress or egress is in
```

Comment:
Update the documentation for this function to reflect the addition of `policy.GetPolicyEnabled()` having its result checked.

Reply:
Done